lib/nitra/workers/rspec.rb

```diff
@@ -139,6 +139,19 @@ def runner_tags(runner)
       (runner.world.all_examples.map(&:metadata).flat_map(&:keys) - STANDARD_METADATA_KEYS).map(&:to_s)
     end
 
+    def filename_without_extension(filename)
```

Comment:
I gues there is no reason to put this in the subclasses. It could live in worker.rb and if the actual concrete type never uses bracketed locations, it would not break anything.